vsintegration/src/FSharp.ProjectSystem.Base/Project/FileNode.cs

```diff
@@ -1070,5 +1070,7 @@ private List<HierarchyNode> GetChildNodes()
             }
             return childNodes;
         }
+
+        public override __VSPROVISIONALVIEWINGSTATUS ProvisionalViewingStatus => __VSPROVISIONALVIEWINGSTATUS.PVS_Enabled;
```

Comment:
Can we make this:
```c#
        public override __VSPROVISIONALVIEWINGSTATUS ProvisionalViewingStatus {
            get
            {
                return IsFileOnDisk(false)
                    ? __VSPROVISIONALVIEWINGSTATUS.PVS_Enabled
                    : __VSPROVISIONALVIEWINGSTATUS.PVS_Disabled;
            }
        }
```

This will fix #2240.